### ISSUE TITLE
Make patterns for Kannada that only break before and after 3 grapheme clusters

### ISSUE THREAD

**Issue by reutenauer** [hyphenation/tex-hyphen#46]
See https://github.com/reutenauer/polyglossia/issues/407#issuecomment-615879308 for context and description.

**Comment by jbezos** [comment]
I wonder if the OP poster is referring to the fact in Kannada words are “hyphenated” showing no hyphens - the words are just broken. I couldn't find a way to get rid of hyphens in XeTeX.

**Comment by reutenauer** [comment]
No, he clearly expects a hyphen, but would like something with the equivalent effect of `\lefthyphenmin=3\righthyphenmin=3`, which doesn’t work well in the Kannada (or other Indic) script because we would need to take grapheme clusters into account, not characters.  I said I would make patterns that mimic that effect.  I think it would be an interesting experiment.

In the mean time the OP said he was happy with horrible space stretching like Word does :-(

**Comment by alaymari** [comment]
Just adding a comment here. In Kannada, hyphens are 'disabled', but words are broken. jbezos is right. I have this snippet in my kannada tex files:

    \def\-{\discretionary{}{}{}}

This gets rid of the `-`, and I can use `\-` to force hyphenation. It avoids all the bad spacing issues.
